.github/scripts/release_images.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Initialize array to store digest SHAs
+declare -a digests
+
+# Fetch all docker image names
+image_names=$(gh api \
+  -H "Accept: application/vnd.github+json" \
+  -H "-GitHub-Api-Version: 2022-11-28" \
+  -H "Authorization: Token ${GITHUB_TOKEN}" \
+  --paginate "/orgs/boozallen/packages?package_type=container" | jq -r '.[] | select((.name | startswith("aissemble")) and (.name | endswith("-chart") | not)) | .name')
+
+# For each docker image, find all release versions, excluding 1.7.0(-arm64/amd64)
+for name in $image_names; do
+  release_versions=$(gh api \
+    -H "Accept: application/vnd.github+json" \
+    -H "-GitHub-Api-Version: 2022-11-28" \
+    -H "Authorization: Token ${GITHUB_TOKEN}" \
+    --paginate "/orgs/boozallen/packages/container/${name}/versions" \
+    | jq -r '.[] | .metadata.container.tags[] | select(test("^\\d+\\.\\d+\\.\\d+?$"))' \
```

Comment:
Is the goal of this regex to match only the release versions (i.e `1.8.0`) but not the SNAPSHOTs (i.e. `1.8.0-SNAPSHOT`)?

If so, ss the `?` on the end of `^\\d+\\.\\d+\\.\\d+?$` necessary? Would think we could just do `^\\d+\\.\\d+\\.\\d+$`

Reply:
yup, you're right! the `?` is no longer needed - updated to `^\\d+\\.\\d+\\.\\d+$`.